src/services/token/token.js

```diff
@@ -0,0 +1,20 @@
+import { ReadOnlyToken, Writeable } from "@govtechsg/oa-token";
+import { getData } from "@govtechsg/open-attestation";
+import { get } from "lodash";
+
+let tokenInstance;
+
+export const initializeToken = async (document, web3Provider = undefined, wallet = undefined) => {
+  tokenInstance = await (web3Provider && wallet
+    ? new Writeable({ document, web3Provider, wallet })
```

Comment:
add logs for the branches here so that we can debug this easily

Reply:
added.